website/docs/r/database_vulnerability_assessment_rule_baseline.html.markdown

```diff
@@ -0,0 +1,159 @@
+  resource_group_name   = "${azurerm_resource_group.test.name}"
+  storage_account_name  = "${azurerm_storage_account.test.name}"
+  container_access_type = "private"
+}
+
+resource "azurerm_template_deployment" "vulnerability_scan_settings" {
+  name = "accteststoragevss"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  deployment_mode = "Incremental"
+
+  parameters = {
+    "sqlServerName"      = "${azurerm_sql_server.test.name}"
+    "storageAccountName" = "${azurerm_storage_account.test.name}"
+    "blobContainerName"  = "${azurerm_storage_container.test.name}"
+  }
+  
+  depends_on = [ "azurerm_sql_database.test" ]
+
+  template_body = <<DEPLOY
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "sqlServerName": {
+      "type": "string"
+    },
+    "storageAccountName": {
+      "type": "string"
+    },
+    "blobContainerName": {
+      "type": "string"
+    }
+  },
+  "resources": [
+    {
+      "apiVersion": "2017-03-01-preview",
+      "type": "Microsoft.Sql/servers/securityAlertPolicies",
+      "name": "[concat(parameters('sqlServerName'), '/DefaultSecurityPolicy')]",
+      "properties": {
+        "state": "Enabled",
+        "emailAddresses": ["support@azure.microsoft.com"],
+        "emailAccountAdmins": false
+      }
+    },
+    {
+      "type": "Microsoft.Sql/servers/vulnerabilityAssessments",
+      "name": "[concat(parameters('sqlServerName'), '/Default')]",
+      "apiVersion": "2018-06-01-preview",
+      "scale": null,
+      "properties": {
+        "storageContainerPath": "[concat('https', '://', parameters('storageAccountName'), '.blob.core.windows.net/', parameters('blobContainerName'), '/')]",
+        "storageAccountAccessKey": "[listKeys(resourceId('Microsoft.Storage/storageAccounts', parameters('storageAccountName')), providers('Microsoft.Storage', 'storageAccounts').apiVersions[0]).keys[0].value]"
+      }
+    }
+  ]
+}
+DEPLOY
+}
+
+resource "azurerm_sql_database" "test" {
+  name                = "mysqldatabase"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  server_name         = "${azurerm_sql_server.test.name}"
+  location            = "${azurerm_resource_group.test.location}"
+  edition             = "Standard"
+}
+
+resource "azurerm_database_vulnerability_assessment_rule_baseline" "test" {
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  server_name = "${azurerm_sql_server.test.name}"
+  database_name = "${azurerm_sql_database.test.name}"
+  rule_id = "VA2065"
+  baseline_name = "master"
+  baseline_result {
+    result = ["allowedip1","123.123.123.123","123.123.123.123"]
+  }
+  baseline_result {
+    result = ["allowedip2","255.255.255.255","255.255.255.255"]
+  }
+  
+
+  depends_on = ["azurerm_template_deployment.vulnerability_scan_settings"]
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
```

Comment:
Could we update the properties to reflect when changing them would create a new resource?

Reply:
done